jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListFilterSettingsManager.java

```diff
@@ -0,0 +1,165 @@
+    }
+
+    @Override
+    public List<FilterSettings> initDefaultFilters() {
+        return Arrays.asList(
+                //Filter status Active
+                createFilterSettings(PROCESS_INSTANCE_DATASET,
+                                     COLUMN_START,
+                                     builder -> builder.filter(equalsTo(COLUMN_STATUS,
+                                                                        ProcessInstance.STATE_ACTIVE)),
+                                     TAB_ACTIVE,
+                                     constants.Active(),
+                                     constants.FilterActive()),
+
+                //Filter status completed
+                createFilterSettings(PROCESS_INSTANCE_DATASET,
+                                     COLUMN_START,
+                                     builder -> builder.filter(equalsTo(COLUMN_STATUS,
+                                                                        ProcessInstance.STATE_COMPLETED)),
+                                     TAB_COMPLETED,
+                                     constants.Completed(),
+                                     constants.FilterCompleted()),
+
+                //Filter status aborted
+                createFilterSettings(PROCESS_INSTANCE_DATASET,
+                                     COLUMN_START,
+                                     builder -> builder.filter(equalsTo(COLUMN_STATUS,
+                                                                        ProcessInstance.STATE_ABORTED)),
+                                     TAB_ABORTED,
+                                     constants.Aborted(),
+                                     constants.FilterAborted())
+        );
+    }
+
+    @Override
+    public FilterSettings getVariablesFilterSettings(final String name) {
+        String tableSettingsJSON = "{\n"
+                + "    \"type\": \"TABLE\",\n"
+                + "    \"filter\": {\n"
+                + "        \"enabled\": \"true\",\n"
+                + "        \"selfapply\": \"true\",\n"
+                + "        \"notification\": \"true\",\n"
+                + "        \"listening\": \"true\"\n"
+                + "    },\n"
+                + "    \"table\": {\n"
+                + "        \"sort\": {\n"
+                + "            \"enabled\": \"true\",\n"
+                + "            \"columnId\": \"" + PROCESS_INSTANCE_ID + "\",\n"
+                + "            \"order\": \"ASCENDING\"\n"
+                + "        }\n"
+                + "    },\n"
+                + "    \"dataSetLookup\": {\n"
+                + "        \"dataSetUuid\": \"jbpmProcessInstancesWithVariables\",\n"
+                + "        \"rowCount\": \"-1\",\n"
+                + "        \"rowOffset\": \"0\",\n";
+        if (name != null) {
+            tableSettingsJSON += "        \"filterOps\":[{\"columnId\":\"" + PROCESS_NAME + "\", \"functionType\":\"EQUALS_TO\", \"terms\":[\"" + name + "\"]}],";
```

Comment:
Are the preceding spaces intentional?

Reply:
to be honest, I'm not sure, just copied the content over from another place, I would love to replace this code with an actual builder instead of string manipulation. Let me try that.